playbooks/images.yaml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Pull images
+  hosts:
+    - quadlet
+  vars_files:
+    - "../vars/images.yml"
+  become: true
+  tasks:
+    - name: Pull an image
+      containers.podman.podman_image:
+        name: "{{ item }}"
+      loop:
+        - quay.io/ehelms/candlepin:4.4.14
```

Comment:
Can we reuse variables? I think we should make sure it doesn't live in 2 places or it will go out of sync

Reply:
:facepalm: Yea, that was the whole reason I created the vars file with images in it. Fixed.